mobile/library/common/jni/jni_interface.cc

```diff
@@ -1109,6 +1112,143 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_registerStringAccessor(JNIEnv*
   return result;
 }
 
+void setString(JNIEnv* env, jstring java_string, EngineBuilder* builder,
```

Comment:
nit: Should these methods be in an anon namespace? (Also, comments would be nice)